internal/inspect/r.go

```diff
@@ -176,15 +176,19 @@ func (i *defaultRInspector) getRVersion(rExecutable string) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	line := strings.SplitN(string(append(output, stderr...)), "\n", 2)[0]
-	m := rVersionRE.FindStringSubmatch(line)
-	if len(m) < 2 {
-		return "", fmt.Errorf("couldn't parse R version from output: %s", line)
-	}
-	version := m[1]
-	i.log.Info("Detected R version", "version", version)
-	rVersionCache[rExecutable] = version
-	return version, nil
+	lines := strings.SplitN(string(append(output, stderr...)), "\n", -1)
+	for _, l := range lines {
+		i.log.Info("Parsing line for R version", "l", l)
```

Comment:
nit: This log might be better as `Debug` instead of `Info`

Reply:
It looks like the existing pattern for this type of tracing within the `r.go` and `python.go` inspection code is `Info`. So I stayed with that to be consistent.